pkgs/tools/audio/beets/default.nix

```diff
@@ -6,52 +6,56 @@
 # Attributes needed for tests of the external plugins
 , callPackage, beets
 
-, enableAcousticbrainz ? true
-, enableAcoustid       ? true
-, enableBadfiles       ? true, flac ? null, mp3val ? null
-, enableConvert        ? true, ffmpeg ? null
-, enableDiscogs        ? true
-, enableEmbyupdate     ? true
-, enableFetchart       ? true
-, enableGmusic         ? true
-, enableKeyfinder      ? true, keyfinder-cli ? null
-, enableKodiupdate     ? true
-, enableLastfm         ? true
-, enableLoadext        ? true
-, enableMpd            ? true
-, enablePlaylist       ? true
-, enableReplaygain     ? true, bs1770gain ? null
-, enableSonosUpdate    ? true
-, enableSubsonicupdate ? true
-, enableThumbnails     ? true
-, enableWeb            ? true
+, enableAcousticbrainz        ? true
+# The closure size will grow rather big if this'll be enabled
```

Comment:
By how much is the closure size increased if it is enabled? Eg. what's the diff between `nix path-info -rs` against enabled/disabled version?

Reply:
TBH, I didn't check that with `nix path-info` when I wrote this. But, I know that the main dependency for this plugin [`essentia`](https://github.com/NixOS/nixpkgs/pull/70389), which I've packaged as well is pretty big. You are invited to look at [it's dependencies](https://github.com/NixOS/nixpkgs/pull/70389/files#diff-d225c5ff5f54e1f2004b2dbf7de01093R52) to get a sense.

Never the less, in a branch where all of the necessary PRs are merged, I've calculated the difference in the closure size by doing the following:

1. Besides my own overlay for beets, which overrides `beets` with `beets = super.beets.override { .. }`, I've created another derivation named `beets-closure-test` with the only difference of this plugin disabled.
2. I ran `nix-build -A beets` and `nix-build -A beets-closure-test`.
3. I ran the following commands and here are the results:

```sh
$ nix path-info -f. -sr beets-closure-test | awk '{ sum += $NF; }END{ print sum; }' | numfmt --to=iec
706M
$ nix path-info -f. -sr beets | awk '{ sum += $NF; }END{ print sum; }' | numfmt --to=iec
977M
```

That's more then a 1/4 of a Gb.